src/resources/js/event-editor.js

```diff
@@ -3,21 +3,143 @@ var tribe_events_event_editor = tribe_events_event_editor || {};
 /**
  * Implements behaviours that are specific to the event editor.
  */
-jQuery( function( $ ) {
-	var obj = tribe_events_event_editor,
-		$sticky_in_month_view_checkbox = $( 'input[name="EventShowInCalendar"]' ),
-		$featured_event_checkbox = $( 'input[name="feature_event"]' );
+( function ( $, obj ) {
+	'use strict';
+
+	obj = obj || {};
+
+	/**
+	 *	Setup our selectors.
+	 *
+	 *	@since TBD
+	 */
+	obj.selectors = {
+		$sticky_in_month_view_checkbox: 'input[name="EventShowInCalendar"]',
+		$featured_event_checkbox: 'input[name="feature_event"]',
+		organizer: {
+			area: '#event_tribe_organizer',
+			delete_button: '.tribe-delete-this',
+			add_button: '#event_tribe_organizer .tribe-add-post',
+			post_dropdown: '.linked-post-dropdown',
+		},
+	};
+
+	obj.organizer = {};
+	/**
+	 *	Controls logic for the Organizer delete button to display.
+	 *
+	 *	If more than 1 organizer exists, display the delete button.
+	 *	If only 1 organizer exists, hide the delete button.
+	 *
+	 *	@since TBD
+	 *	@return void
+	 */
+	obj.organizer.deleteButtonDisplayLogic = () => {
+
+		const $organizers = $( obj.selectors.organizer.area ).find( '.saved-linked-post' );
+
+		$( obj.selectors.organizer.area )
+			.find( obj.selectors.organizer.delete_button )
+			.each( function () {
+
+				// If you have more than 1 organizer then we display the delete button.
+				if ( $organizers.length > 1 ) {
+					$( this ).show();
+					return;
+				}
+
+				const $organizer_dropdown = $( obj.selectors.organizer.area + ' ' + obj.selectors.organizer.post_dropdown );
+
+				//If this is running, it's because we only have 1 organizer
+				if ( $organizer_dropdown.val() !== '-1' ) {
+					$( obj.selectors.organizer.add_button ).show();
+				}
+
+				$( this ).hide();
+
+			} );
+
+	};
+	/**
+	 *	Trigger events for bind events.
+	 *
+	 *	@since TBD
+	 */
+	obj.organizer.bindEvents = () => {
+
+		/**
+		 *	Logic for the organizer area -
+		 *	"Add Organizer" button should be hidden by default,
+		 *	only appearing when there is more than 1 organizer.
+		 *
+		 *	"Trash" icon / delete button should be hidden by default,
+		 *	only appearing when there is more than 1 organizer
```

Comment:
```suggestion
		 *	only appearing when there is at least one organizer
```

Reply:
TBH not sure about this logic - do we **not** want to show the trash icon for the first/only organizer?

Reply:
I have the code setup so that when only one organizer is selected no delete icon will appear. As soon as a second organizer is selected the delete icon will appear for all organizers. 

The thought process behind this is that when a single organizer is selected it is better to "reset" the dropdown than to allow them to delete the table row via the delete icon.

Reply:
OK - the ticket is notably lacking in AC 😢 so I wasn't sure what was expected. Carry on 🖖🏼